test/utils/test-constants.js

```diff
@@ -3,8 +3,8 @@
 var path = require('path');
 
 // Input/output relative paths
-var inputRelative = './fixtures/';
-var outputRelative = './out-fixtures/';
+var inputRelative = './fixtures';
```

Comment:
Alright, I didn't look at the test titles to see if any of them mentioned "ending in a separator" which would be incorrect titles now.